README.md

```diff
@@ -6,3 +6,6 @@ This repository contains starter projects completed as part of the STEP
 internship program. Right now, this includes a personal portfolio built 
 for Google Cloud's app engine -- build and deployment instructions are
 contained in `portfolio/`.
+
+The portfolio is deployed here: 
+[https://codyjrivera-step-2020.uc.r.appspot.com](https://codyjrivera-step-2020.uc.r.appspot.com)
```

Comment:
optional: If you want to link the URL as is in the markdown I don't _think_ you need to hyperlink it.

Reply:
I will remove it.